python-spec/src/somacore/ephemeral/collections.py

```diff
@@ -154,6 +164,43 @@ def transformations(self) -> MutableMapping[str, coordinates.CoordinateTransform
         raise NotImplementedError()
 
 
+class Image2D(  # type: ignore[misc]   # __eq__ false positive
```

Comment:
Since this are explicitly multiscale, I think it would make sense if that was clear in the name. Maybe `MultiscaleImage2D`? I think there's a version of the spec where these were called `ImageCollection`?